lineage/figures/figure2.py

```diff
@@ -1,22 +1,18 @@
 """
 This creates Figure 2.
+Should be the model figure.
 """
 from .figureCommon import subplotLabel, getSetup
-from .Matplot_gen import Matplot_gen
-from .Fig_Gen import Lineages_per_Population_Figure
+
 
 
 def makeFigure():
```

Comment:
```suggestion
def makeFigure2():
```

Reply:
Actually, don't think this is recommended. This might break something with the way Dr. Meyer has it set up. Ignore for now.